org.gnome.SimpleScan.json

```diff
@@ -196,14 +196,18 @@
                 "--with-cupsbackenddir=/app/lib/cups/backend"
             ],
             "build-options": {
-                "cppflags": "-I/app/include/libusb-1.0"
+                "cppflags": "-I/app/include/libusb-1.0 -Wno-error=implicit-function-declaration -Wno-error=return-mismatch -Wno-error=incompatible-pointer-types"
```

Comment:
This is certainly not the correct fix, there is a problem with symbols/headers being gone and needs to be debugged. This very likely explodes at runtime atm